pulsar/internal/rpc_client.go

```diff
@@ -64,6 +64,9 @@ type RPCClient interface {
 	RequestToHost(serviceNameResolver *ServiceNameResolver, requestID uint64,
 		cmdType pb.BaseCommand_Type, message proto.Message) (*RPCResult, error)
 
+	RequestWithCnxKeySuffix(logicalAddr *url.URL, physicalAddr *url.URL, cnxKeySuffix int32, requestID uint64,
```

Comment:
@shibd You can change the `Request` method to avoid introducing an unnecessary method because the `PRCClient` is an internal interface. I checked the usage of the `Request` method and found one call now.

Reply:
Lookup will use this method, And separate them to make the meaning of the two methods more direct.